File: EYImageSliderDemo-OC/EYImageSliderDemo-OC/EYImageSliderStoryboard.m
//
//  EYImageSliderStoryboard.m
//  EYImageSliderDemo-OC
//
//  Created by Enix Yu on 29/10/2016.
//  Copyright © 2016 RobotBros. All rights reserved.
//

#import "EYImageSliderStoryboard.h"

@interface EYImageSliderStoryboard ()

@property (weak, nonatomic) IBOutlet EYImageSlider *imageSlider;

@end

@implementation EYImageSliderStoryboard


- (void)viewDidLoad
{
    [super viewDidLoad];
    
    [self.imageSlider setDataSource:self];
    [self.imageSlider setDelegate:self];
    
    [self.imageSlider setSlideShowTimeInterval:2.0];
    [self.imageSlider setCaptionTextColor:[UIColor whiteColor]];
}

@end

File: EYImageSliderDemo/EYImageSliderDemo/EYImageSliderStoryboard.swift
//
//  ViewController.swift
//  EYImageSliderDemo
//
//  Created by Enix Yu on 29/10/2016.
//  Copyright © 2016 RobotBros. All rights reserved.
//

import UIKit
import EYImageSlider

class EYImageSliderStoryboard: EYImageSliderBase {

    @IBOutlet weak var imageSlider: EYImageSlider!
    
    override func viewDidLoad() {
        super.viewDidLoad()
        
        // Set slider data source & delegate
        imageSlider.dataSource = self
        imageSlider.delegate = self
        
        // Slider show time interval
        imageSlider.slideShowTimeInterval = 3
        
        // Set image caption
        imageSlider.captionTextColor = UIColor.whiteColor()
    }
}

